SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -807,39 +809,50 @@ void USpatialActorChannel::RemoveRepNotifiesWithUnresolvedObjs(TArray<UProperty*
 	});
 }
 
-void USpatialActorChannel::SpatialViewTick()
+void USpatialActorChannel::ServerCheckIfOwnershipChanged()
```

Comment:
👍 good name

Reply:
Perhaps these function names should be `ProcessOwnershipChange` ?

Reply:
Yep, that sounds better.